settings.py

```diff
@@ -21,6 +21,6 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = True
```

Comment:
Ensure that the `USE_TZ` setting aligns with your application's requirements. If your application needs to handle time zones differently, you may need to adjust this setting.